### DISCUSSION TITLE
Heliarch Anti-Missile

### DISCUSSION THREAD

**Discussion by SultanRTG** [endless-sky/endless-sky#9306] [Ideas]
I checked on the wiki, and found out that there is no current Coalition/Heliarch Anti-Missile turret. This seems a little weird, because they have very strong missiles. Maybe there could be a part in the Heliarch storyline where a rebellion movement starts using missiles-you have to go and buy various anti-missile turrets (Human, Hai, Wanderer, Korath, Remnant). This results in the creation of a Heliarch Anti-Missile turret (maybe new ship variants that have one more turret slot to put Anti-Missile turrets on). There could possibly be a difference on the anti missile based on which turret you give them: Hai and Korath would go for more shots per second, Wanderer and Remnant results in less shots but stronger blasts (with more energy used), and Human would go for a mix of the two (like how you decide to pick more weapon slots, engine space, or shields for the Kestrel).

**Comment by lumbar527** [comment]
I'm not sure about that; there might to be something in #6646 about getting the Lunarium (Resistance) some human anti-missile. I'm not sure an am not going to look through that now.

**Reply by Arachi-Lover** [reply]
The plan is for the Lunarium/Resistance to develop AM of their own (based on the human ones) to counter the Heliarch torpedoes yes.

The Heliarchs won't have AM, as their kit is entirely designed to fight the Quarg.
